pages/docs/user-docs/build-shub.md

```diff
@@ -0,0 +1,28 @@
+---
+title: shub bootstrap module
+sidebar: user_docs
+permalink: build-shub
+folder: docs
+toc: false
+---
+
+This module allows you to build a container from a container hosted on Singularity Hub. 
+
+{% include toc.html %}
+
+## Overview
+You can use an existing container on Singularity Hub as your "base," and then add customization. This allows you to build multiple images from the same starting point. For example, you may want to build several containers with the same custom python installation, the same custom compiler toolchain, or the same base MPI installation. Instead of building these from scratch each time, you could create a base container on Singularity Hub and then build new containers from that existing base container adding customizations in `%post`, `%environment`, `%runscript`, etc.
+
```

Comment:
I sense repeated content from the local image :) Maybe mix it up a bit?

Reply:
whatevs.  This isn't supposed to be interesting it's supposed to be informative.  And we are in a time crunch.